src/operator/src/insert.rs

```diff
@@ -281,10 +287,11 @@ impl Inserter {
             table_infos,
         } = self
             .create_or_alter_tables_on_demand(
-                &requests,
+                &mut requests,
                 &ctx,
                 AutoCreateTableType::Logical(physical_table.to_string()),
                 statement_executor,
+                true,
```

Comment:
Not quite sure, but the influxdb line protocol will go into this function too?

Reply:
No, only PromStoreProtocolHandler will use this

Reply:
But the influxdb needs this fix too.

Reply:
added